pubspec.yaml

```diff
@@ -13,11 +13,15 @@ dependencies:
     sdk: flutter
   intl: any
   cupertino_icons: ^1.0.5
+  built_collection: ^5.1.1
+  built_value: ^8.6.0
```

Comment:
Can we make it without introducing code generation or data class for V1?
I think it can be complex for newbies. I'd like to make it simple for the V1.

Reply:
Also, out of curiosity. Are these packages better than freezed?